plugin.yaml

```diff
@@ -42,10 +37,6 @@ node_types:
           https://cloud.google.com/compute/docs/machine-types
         type: string
         default: n1-standard-1
-      instance_additional_settings:
-        description: >
-          Additional instance settings.
-        default: {}
```

Comment:
Adding 'additional settings' that will update the body dictionary to each resource will make the plugin more flexible if any other attributes of a resource will be supported in the future or we don't support some yet (https://github.com/Cloudify-PS/cloudify-gcp-plugin/commit/9c01ccb92262ce238ed2f4dc0ec8db112fd105b4). What do you think about it?